### DISCUSSION TITLE
vncserver help please.

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#16696] [Q&A] [answered]
Strange thing happening.
I cannot start a vncserver on any port lower than 5905. No sockets in termux tmp directory.
`vncserver :1
A VNC server is already running as :1
u0_a247@localhost:~$ cd ../usr/tmp
u0_a247@localhost:.../usr/tmp$ ls -al
total 16
drwx------  2 u0_a247 u0_a247 4096 May 19 10:52 .
drwx------ 11 u0_a247 u0_a247 4096 Nov  3  2022 ..`  Works fine on 5906 and above.
Just curious why that is.
Help much appreciated.

**Comment by sylirre** [comment, marked as answer]
Force stop Termux app and open it again.

Look `vncserver -list` output. The VNC server keeps own list of instances which doesn't always match the available sockets. You need to `vncserver -kill` them.

This happens because VNC server was not gracefully terminated.

**Reply by macerg** [reply]
No sessions.

`TigerVNC server sessions:
                                                        X DISPLAY #     PROCESS ID`

**Reply by macerg** [reply]
Yes, worked ok after I force closed termux.
Thanks 🙏👍

**Reply by macerg** [reply]
Seems to be related to me running autossh for my ssh tunnels.

**Reply by macerg** [reply]
When I shut the SSH tunnels down, the 'busy' ports are freed up.

**Comment by macerg** [comment]
Closed, thanks.
